q2_feature_classifier/_consensus_assignment.py

```diff
@@ -242,3 +243,9 @@ def _compute_consensus_annotation(annotations, min_consensus,
         consensus_fraction_result = 1.0
 
     return annotation, consensus_fraction_result
+
+
+def _annotate_method(taxa, method):
```

Comment:
I wish we had some way to ship "hidden" actions on plugins, this utility would be a great candidate for that (without it we break provenance). Add that one to the TODO list...

Reply:
yes! we've had this discussion before re: actions in q2-sample-classifier and q2-longitudinal that should be "hidden". I suppose `annotate_method` could be usefully exposed as a method, allowing users to reconstruct their own hybrid classifier pipelines...

Reply:
> I suppose annotate_method could be usefully exposed as a method

Your call :call_me_hand:

Reply:
let's leave it hidden for now